### ISSUE TITLE
Required object attribute(s) are missing from your request: cell.value

### ISSUE THREAD

**Issue by sun-pgdx** [smartsheet-platform/smartsheet-python-sdk#108]
Hello,

Our software is attempting to insert 366 new rows into the target Smartsheet.

Upon invoking the SDK (smartsheet-python-sdk 1.3.3) add_rows() method, the error response below is returned from Smartsheet.

How should we interpret the following error response?

Does the detail.index suggest that it was the 5th row (record) that was missing a value for one of the cells?  Which cell?  The first one?

Is there a way to turn up the verbosity so that the error response will indicate which of the 366 inbound rows contain missing values and which cells specifically?

Thank you in advance.

<pre>
Response: {
  status: 400 Bad Request
  content: {
    {
    "detail": {
        "index": 5
    },
    "errorCode": 1012,
    "message": "Required object attribute(s) are missing from your request: cell.value.",
    "refId": "1ayar3kc6daio"
  }
}
</pre>

**Comment by timwellswa** [comment]
Yes, that is correct. The 5th row is the location of the error. (https://smartsheet-platform.github.io/api-docs/#add-rows).

Unfortunately, there is no more error detail available. For the rows that fail you are going to have to look at each of the cells individually to determine which one failed (sorry, I don't have a better answer). If you turn the logging on it should give you a request body dump that should allow you to look into more detail for the failing rows.

You can also try add_rows_with_partial_success if you want add_rows to continue even if it encounters an error (the partial success return result will contain an array containing the indices of rows that failed - if any).

**Comment by Paulq19** [comment]
I got same problem, looks no idea how pass through
